src/api/app/views/layouts/webui/responsive_ux/_sponsors.html.haml

```diff
@@ -0,0 +1,7 @@
+- if CONFIG['sponsors'].present?
+  .sponsors-header Open Build Service is sponsored by
```

Comment:
This should be the `Configuration.title`

Reply:
Fixed